github3/repos/users.py

```diff
@@ -0,0 +1,33 @@
+from ..users import _User
```

Comment:
Is there reason to not just put these classes in `github3.users`?

Reply:
No reason at all, preference perhaps. It'll get changed.

Reply:
Oh wait, now that I think about it. The reason I put it in `github3.repos.users` instead of `github3.repos.users` is because I assumed that was implied in the issue in the issue #670. As a collaborator's endpoint is `/repos/:owner/:repo/collaborators`. They belong to the repo so I thought it would be good to put those in `github3.repos.users` instead. Regardless though, tell me what you think is best.